homework 1

F1-R3 opened the pull request.

kellolo commented:
Не тот репозиторий форкнули..........
Вот мой: https://github.com/kellolo/R-prof-5-2908

AlinaNasibullina committed:
Merge pull request #4 from kellolo/master

lesson4

Roman111975 committed:
Home_Lesson3

xorrou1 committed:
Merge pull request #4 from kellolo/master

merge

Roman111975 committed:
HW_Lesson_4

ksu21 committed:
Merge pull request #3 from kellolo/master

4 lesson

Commit:
Denis Zverev(3)

Alaya95 committed:
New files shablonstext

eqillibrium committed:
Merge pull request #3 from kellolo/master

Pull from kelolo's master branch

eqillibrium committed:
new commit (create validate form)

eqillibrium committed:
new commit (project refactoring)

HuzinRuslan committed:
Merge pull request #4 from kellolo/master

update_4

Commit:
Denis Zverev(4)

kellolo committed:
Merge pull request #71 from HuzinRuslan/master

Ruslan_lesson_4

kellolo committed:
Merge pull request #70 from eqillibrium/master

HM#4

kellolo committed:
Merge pull request #69 from Celestialis/master

Denis Zverev(4)

kellolo committed:
Merge pull request #68 from Roman111975/master

Home_Lesson3

kellolo committed:
Merge pull request #67 from VeronShat/master

Занятие 2: добавила плагины

kulyamzin committed:
extends basket + catalog

annaelody committed:
Merge pull request #3 from kellolo/master

2

AlinaNasibullina committed:
Merge pull request #5 from kellolo/master

lesson 4

AlinaNasibullina committed:
HW4 other folder

Veledar committed:
Практическое задание Урок 4. Регулярные выражения

ast-webdeveloper committed:
Merge pull request #6 from kellolo/master

updating the repository

astarasov79 committed:
Creating working classes and a working build with dev, without styling the basket.

GeorgeCybersport committed:
homework done

Alaya95 committed:
New files form

Alaya95 committed:
modified

Alaya95 committed:
modified

Alaya95 committed:
Merge pull request #4 from kellolo/master

merge3

xorrou1 committed:
Merge pull request #5 from kellolo/master

merge

xorrou1 committed:
Добавил в поэкт наследуемые классы List и Items

xorrou1 committed:
Добавил в поэкт наследуемые классы List и Items' наследуемые классы List и Items

ksu21 committed:
4th hm

RomiusGod committed:
Через промис запрос и переработку jсына реализовал.
Найти общее в корзине и каталоге не смог, чтоб не сломав из них вывести общий класс

kellolo committed:
5

kellolo committed:
Merge branch 'master' of https://github.com/kellolo/kellolo-js-2-21-1209

Celestialis committed:
Merge pull request #1 from kellolo/master

Подкачка 5 урока

ksu21 committed:
Merge pull request #4 from kellolo/master

5 lesson

GeorgeCybersport committed:
Merge pull request #1 from kellolo/master

7епангн

kellolo committed:
Merge pull request #81 from RomiusGod/master

ДЗ для 3 урока, где разбирали промисы.

kellolo committed:
Merge pull request #80 from annaelody/master

homework_2

kellolo committed:
Merge pull request #79 from ksu21/master

4 hm

kellolo committed:
Merge pull request #78 from kulyamzin/master

Extends basket + catalog

kellolo committed:
Merge pull request #77 from xorrou1/master

Добавил в проект наследуемые классы 'List и Items'

kellolo committed:
Merge pull request #76 from Alaya95/master

hw4

kellolo committed:
Merge pull request #75 from GeorgeCybersport/master

homework done

kellolo committed:
Merge pull request #74 from Tarasov-Andrey-student/master

Creating working classes and a working build with dev, without styling the basket.

kellolo committed:
Merge pull request #73 from Veledar/master

ПЗ Урок 3

kellolo committed:
Merge pull request #72 from AlinaNasibullina/master

HW4

eqillibrium committed:
Merge pull request #4 from kellolo/master

Pull from kelolo's master branch

Celestialis committed:
Merge pull request #2 from kellolo/master

2

xorrou1 committed:
Создал файл vue.js. Не получается сделать запрос

eqillibrium committed:
new commit

Commit:
Denis Zverev

Commit:
Denis Zverev

Commit:
Denis Zverev

kulyamzin committed:
regularExpression homework

HuzinRuslan committed:
Merge pull request #5 from kellolo/master

update_5

HuzinRuslan committed:
vue_1

Veledar committed:
Merge pull request #5 from kellolo/master

Вытянул после 5 урока

AlinaNasibullina committed:
Merge pull request #6 from kellolo/master

lesson5

Roman111975 committed:
Lesson_5

Alaya95 committed:
add vue file

Veledar committed:
Заглушка поле 5 урока

kulyamzin committed:
Merge pull request #1 from kellolo/master

update from teacher branch

ksu21 committed:
5th hm

kellolo committed:
Merge pull request #88 from ksu21/master

5th hw

kellolo committed:
Merge pull request #86 from Roman111975/master

Lesson_5

kellolo committed:
Merge pull request #85 from HuzinRuslan/master

Ruslan_lesson_5

kellolo committed:
Merge pull request #84 from Celestialis/master

Denis Zverev

kellolo committed:
Merge pull request #83 from eqillibrium/master

HM#5

kellolo committed:
Merge pull request #82 from xorrou1/master

Создал файл vue.js. Не получается сделать запрос с сервера

kulyamzin committed:
webapack version update

AlinaNasibullina committed:
hw5, add new Vue

Alaya95 committed:
add vue for vue

Alaya95 committed:
add button for cart

Alaya95 committed:
Merge pull request #5 from kellolo/master

merge

Alaya95 committed:
Merge branch 'master' of github.com:Alaya95/kellolo-js-2-21-1209

kulyamzin committed:
regularExpression in others + vue catalog

GeorgeCybersport committed:
add vue

VeronShat committed:
ДЗ к уроку 3

Теперь кнопка add работает и добавляет в корзину

kellolo committed:
Merge pull request #93 from VeronShat/master

ДЗ к уроку 3

kellolo committed:
Merge pull request #92 from GeorgeCybersport/master

add vue

kellolo committed:
Merge pull request #91 from kulyamzin/master

regExp + Vue catalog on index

kellolo committed:
Merge pull request #90 from Alaya95/master

hw5 add vue

kellolo committed:
Merge pull request #89 from AlinaNasibullina/master

HW5

HuzinRuslan committed:
Merge pull request #6 from kellolo/master

update_5.1

HuzinRuslan committed:
questionFormOn

kellolo committed:
hw + add 6 project

kellolo committed:
6

Veledar committed:
Merge pull request #6 from kellolo/master

После 6 Урока

Veledar committed:
Классы - добавил item и list

ksu21 committed:
Merge pull request #5 from kellolo/master

6 lesson

kellolo committed:
Merge pull request #94 from HuzinRuslan/master

Включил форму

Celestialis committed:
Merge pull request #3 from kellolo/master

6

xorrou1 committed:
Merge pull request #6 from kellolo/master

merge

kulyamzin committed:
changed styles

Roman111975 committed:
HW_Lesson_6

eqillibrium committed:
Merge pull request #5 from kellolo/master

Pull from master

HuzinRuslan committed:
vue_2

HuzinRuslan committed:
Merge pull request #7 from kellolo/master

update_6

Alaya95 committed:
install vue, add Main.vue

eqillibrium committed:
new commit

Alaya95 committed:
install Vue

Commit:
Denis Zverev

Commit:
Denis Zverev

AlinaNasibullina committed:
Merge pull request #7 from kellolo/master

lesson6

Veledar committed:
Классы с наследованием

xorrou1 committed:
Перевел проэкт на пробный VUE

kellolo committed:
Merge pull request #98 from xorrou1/master

Перевел проект на пробный VUE

kellolo committed:
Merge pull request #97 from Celestialis/master

Denis Zverev

kellolo committed:
Merge pull request #96 from HuzinRuslan/master

Ruslan_lesson_6

kellolo committed:
Merge pull request #95 from Roman111975/master

HW_Lesson_6

Veledar committed:
Поменял <a> на <button> в каталоге

Alaya95 committed:
Add vue file

Alaya95 committed:
removed the html structure and added it to main

Alaya95 committed:
removed js files and added to vue

Veledar committed:
Не рабочая регулярка с Урока 4

kulyamzin committed:
changed some styles

eqillibrium committed:
new commit

Alaya95 committed:
Vue

Alaya95 committed:
Merge pull request #6 from kellolo/master

merge

Alaya95 committed:
Merge branch 'master' of github.com:Alaya95/kellolo-js-2-21-1209

Oleg166 committed:
Выполнил ДЗ к 4-ой лекции - папка /other/dz4
Выполнил ДЗ к 5-ой лекции - по видео с вебинара

Oleg166 committed:
Merge pull request #1 from kellolo/master

After 6 lesson

Oleg166 committed:
ДЗ к 4 лекции - в папке Oleg Dementev/other/dz4
ДЗ к 5 лекции - в папке Oleg Dementev/Project (повтор того, как сделали на вебинаре)

AlinaNasibullina committed:
Merge pull request #8 from kellolo/master

other

AlinaNasibullina committed:
hw6, add vue components, add  computed total sum, moved imgs to github

GeorgeCybersport committed:
Merge pull request #2 from kellolo/master

1

GeorgeCybersport committed:
add components

Veledar committed:
Vue начало - козина не работает

eqillibrium committed:
new commit (HM#6 is finished)

Veledar committed:
Vue в HTML

Veledar committed:
Vue в index.js лишнее убрал. Корзина идеальная

ksu21 committed:
6th hw

RomiusGod committed:
Merge pull request #3 from kellolo/master

Update project

Veledar committed:
Vue Catalog Basket Сборка

VeronShat committed:
Домашка 4 урока

Домашка к 4 уроку в other и наследование

Veledar committed:
Vue Не удачная попытка подсчитать корзину

kulyamzin committed:
add basket.vue + catalog.vue

kulyamzin committed:
improved vue

kellolo committed:
Merge pull request #103 from eqillibrium/master

HM#6 is finished

kellolo committed:
7

kellolo committed:
7.001

eqillibrium committed:
Merge pull request #6 from kellolo/master

pull from master

ksu21 committed:
Merge pull request #6 from kellolo/master

7 lesson

Celestialis committed:
Merge pull request #4 from kellolo/master

7

eqillibrium committed:
copied server from lesson

AlinaNasibullina committed:
Merge pull request #9 from kellolo/master

lesson 7

Oleg166 committed:
Merge pull request #2 from kellolo/master

Обновление репозитория после 7-го урока

Commit:
Denis Zverev

HuzinRuslan committed:
server_1

HuzinRuslan committed:
Merge pull request #8 from kellolo/master

update_7

ast-webdeveloper committed:
Merge pull request #7 from kellolo/master

updating the repository

Alaya95 committed:
add basket.json

Alaya95 committed:
add catalog.json

Alaya95 committed:
add package

GeorgeCybersport committed:
Merge pull request #3 from kellolo/master

123

AlinaNasibullina committed:
add server

Roman111975 committed:
Lesson_7

RomiusGod committed:
Так как у самого ничего не вышло, повторил с преподавательской версии почти всё, в том числе и картинки.

AlinaNasibullina committed:
HW7

astarasov79 committed:
homework done for lesson 5-6

Alaya95 committed:
add index.js

Alaya95 committed:
add link

Alaya95 committed:
Merge pull request #7 from kellolo/master

merge

Alaya95 committed:
Merge branch 'master' of github.com:Alaya95/kellolo-js-2-21-1209

astarasov79 committed:
completing homework for lesson 7

astarasov79 committed:
completing homework for lesson 7

Veledar committed:
Merge pull request #7 from kellolo/master

После 7 урока

Veledar committed:
Заглушка после 7 урока

kellolo committed:
8

Veledar committed:
Merge pull request #8 from kellolo/master

8

Celestialis committed:
Merge pull request #5 from kellolo/master

8

AlinaNasibullina committed:
Merge pull request #10 from kellolo/master

8

HuzinRuslan committed:
server_2

HuzinRuslan committed:
Merge pull request #9 from kellolo/master

update_8

Commit:
Denis Zverev

GeorgeCybersport committed:
Merge pull request #4 from kellolo/master

8

eqillibrium committed:
Merge pull request #7 from kellolo/master

8

Roman111975 committed:
Lesson_8

Veledar committed:
Откат и Востановление

Veledar committed:
Vue один Item для корзины и каталога - Работает

Veledar committed:
Уд. из корз. переделал на пользовательское событие

Veledar committed:
Server начало, после установки Ошибка

ast-webdeveloper committed:
Merge pull request #8 from kellolo/master

updating the repository

kellolo committed:
Merge pull request #116 from Tarasov-Andrey-student/master

completing homework for lesson 7

kellolo committed:
Merge pull request #115 from Alaya95/master

hw 7

kellolo committed:
Merge pull request #114 from AlinaNasibullina/master

HW6, HW7

kellolo committed:
Merge pull request #113 from RomiusGod/master

Много файлов(Все мои). Недоделанное ДЗ, буду продвигаться дальше позднее. Пока то, что успел.

kellolo committed:
Merge pull request #112 from Roman111975/master

Lesson_7

kellolo committed:
Merge pull request #111 from HuzinRuslan/master

Ruslan_lesson_7

kellolo committed:
Merge pull request #110 from Celestialis/master

Denis Zverev(7)

kellolo committed:
Merge pull request #109 from Oleg166/master

ДЗ к 6 и 7 лекциям

kellolo committed:
Merge pull request #108 from eqillibrium/master

copied server from lesson

kellolo committed:
Merge pull request #107 from kulyamzin/master

catalog.vue + basket.vue

kellolo committed:
Merge pull request #106 from Veledar/master

Времени не хватило разобраться как корзина считается

kellolo committed:
Merge pull request #105 from VeronShat/master

Домашка 4 урока

kellolo committed:
Merge pull request #104 from ksu21/master

6th hw

kellolo committed:
Merge pull request #102 from GeorgeCybersport/master

добавил компаненты.

Veledar committed:
SERVER база готова, работает

Veledar committed:
запуск servera

Veledar committed:
Получение данных JSON

Veledar committed:
Front претворяется 3000, получение данных с server

Veledar committed:
Подготовка к Билду с урока 7

GeorgeCybersport committed:
final project

eqillibrium committed:
end native project

xorrou1 committed:
Merge pull request #7 from kellolo/master

merge

xorrou1 committed:
Пересобрал на vue, пока не работает

xorrou1 committed:
Merge branch 'master' of github.com:xorrou1/kellolo-js-2-21-1209 into master

astarasov79 committed:
the build works with dev, but images don_t load

Alaya95 committed:
add new style

Alaya95 committed:
compile scss style

Alaya95 committed:
modified style

Alaya95 committed:
style

Alaya95 committed:
add nav vue

Alaya95 committed:
add hr

Alaya95 committed:
replacing a link

astarasov79 committed:
creating a project on vue-cli

ast-webdeveloper committed:
Merge pull request #9 from kellolo/master

updating the repository

astarasov79 committed:
Merge branch 'master' of https://github.com/Tarasov-Andrey-student/kellolo-js-2-21-1209 into master

astarasov79 committed:
creating a server

Alaya95 committed:
final project v 1.0

Alaya95 committed:
Merge pull request #8 from kellolo/master

merge

Alaya95 committed:
Merge branch 'master' of github.com:Alaya95/kellolo-js-2-21-1209

eqillibrium committed:
Final cli-project in progress

Oleg166 committed:
Выполнил задание к 6-ой лекции.
По заданию к 7-ой лекции сделал, чтобы данные по корзине (basket) загружались также, как и для каталога (из файла формата json с сервера)

kellolo committed:
Merge pull request #121 from Alaya95/master

final project v 1.0

kellolo committed:
Merge pull request #120 from Tarasov-Andrey-student/master

building a project on vue-cli

kellolo committed:
Merge pull request #119 from Veledar/master

Продолжение следует...

kellolo committed:
Merge pull request #118 from xorrou1/master

Сдаю урок 6

kellolo committed:
Merge pull request #117 from GeorgeCybersport/master

final project

Veledar committed:
Merge pull request #9 from kellolo/master

подтянул после принятия 7 урока

eqillibrium committed:
vuex is finished

eqillibrium committed:
cli-project beta

Veledar committed:
Фронт - разделил на компоненты, без <header>

Veledar committed:
Фронт - переделал компоненты оставил только нужные

Veledar committed:
перекинул сборку с Фронта на Сервер 3000

Veledar committed:
Добавление/Удаление - Корзина. Странности с Id

Veledar committed:
Добав-е не имеющегося товара в корзину на сервер

Veledar committed:
POST запрос полностью готов, добавляется/удаляется

Veledar committed:
Vue cli + доп.компоненты + настройка - BASE

Veledar committed:
Перенёс свой проект во Vue cli + настройки

Veledar committed:
Vue-cli работает кроме добавление товара в корзину

Veledar committed:
Исправил мелкие ошибки

Veledar committed:
Vue-cli + подготовка к SPA

Veledar committed:
Установка тестировщика jasmine

Veledar committed:
Корзина считает, но реактивность не работает

kellolo committed:
Merge pull request #124 from Veledar/master

Final 8.003

kellolo committed:
Merge pull request #122 from eqillibrium/master

cli-project with vuex
